qiita_ware/commands.py

```diff
@@ -70,22 +71,41 @@ def submit_EBI(preprocessed_data_id, action, send, fastq_dir_fp=None):
     ebi_submission.write_xml_file(ebi_submission.generate_submission_xml(),
                                   ebi_submission.submission_xml_fp)
 
-    # other steps
     if send:
-        ebi_submission.send_sequences()
-        study_accession, submission_accession = ebi_submission.send_xml()
+        # step 4: sending sequences
+
+        # Set the ASCP password to the one in the Qiita config, but remember
+        # the old pass so that we can politely reset it
+        old_ascp_pass = environ.get('ASPERA_SCP_PASS', '')
```

Comment:
```python
LogEntry.create('Runtime', "Submitting sequences for pre_processed_id: %d" % preprocessed_id)`
try:
    # place holder for moi call see #1477
    pass 
except:
    # handle exception
    LogEntry.create('Fatal', ...
else:
    LogEntry.create('Runtime', 'Submission of sequences of pre_processed_id: %d completed successfully' % preprocessed_id)
...
```

This allows us to get timing information for free, and to verify that events actually happened. In principle, logging all events that happen is useful particularly as it isn't clear what will and will not be useful in the future. Take, for example, the default verbosity of virtually any server like Apache, nginx, Tornado, vsftp, dhcpd, Torque, slurm, postfix, sendmail, etc.